docs/quality-assurance-test-cases.md

```diff
@@ -173,7 +173,29 @@ This document contains some examples of test cases for each feature implemented
 | Logged in user tries to complete a task from an non existing request (The request  exists in a different relationship) | Fail |
 | Logged in user tries to complete a task from a request which is not in the ACCEPTED state (as a mentor or as a mentee) | Fail |
 
+### List All Tasks
+**Service:** PUT /mentorship_relation/{request_id}/task/{task_id}
```

Comment:
This is still the wrong endpoint. 
If you have doubts, you can always refer the docs.
i.e for this case : **GET `/mentorship_relation/{request_id}/tasks`** will be the endpoint.
As you can see on https://mentorship-backend-temp.herokuapp.com/

Check the first endpoint in below image.

![Mentorship](https://user-images.githubusercontent.com/18344707/92134200-df417b80-ee26-11ea-8ba5-68610937111f.png)